src/mappings/DAOTracker/datasource.yaml

```diff
@@ -0,0 +1,20 @@
+abis:
+  - DAOTracker
+entities:
+  - DAOTrackerContract
+templates:
+  - mapping: Avatar
+    version: 0.0.1-rc.31
+  - mapping: Controller
+    version: 0.0.1-rc.31
+  - mapping: DAOToken
+    version: 0.0.1-rc.31
+  - mapping: Reputation
+    version: 0.0.1-rc.31
```

Comment:
where the schemes templates (e.g GenericScheme) is added ?

Reply:
The scheme contracts do not need to be templates because their addresses are known at startup. We cannot support dynamically tracking schemes because we don't currently have a way of determining the "Scheme Type" when a new scheme is added to the controller. We will once Arc Factories are a thing though.

Reply:
well, you can determining the `Scheme Type` via contractInfo , via its address